_posts/2022-09-13-leader-election.md

```diff
@@ -0,0 +1,15 @@
+---
+layout: post
+title:  "Video: Leader Election - Running Cluster Operator with multiple replicas"
+date: 2022-09-13
+author: jakub_scholz
+---
+
+Leader Election is one of the new features in Strimzi 0.31.0.
+In our new video, we look at it in more detail.
+We explain how it works, when is it useful and show a demo of Leader Election in action.
```

Comment:
"when it is useful" ... and "shows a demo" ?

Reply:
Why _shows a demo_? It is _We show a demo_ so I think that is correct.

Reply:
@PaulRMellor Can you please have a look?

Reply:
No no you are right :-D ... I read as "it shows", my bad. Only thin I am not sure if for a native English it needs a reinforcement of "We" again because the verb is so far from the subject. But it's something for @PaulRMellor :-D

Reply:
Let's just use demo as a verb ;-)